setup/azurecmdfiles.wxi

```diff
@@ -3984,9 +3984,6 @@
                         <Component Id="cmpF7ADD37722F45985B5323202F35958F2" Guid="*">
                             <File Id="fil0FA91D9C31AF07DCD03BFB842FE2B77E" KeyPath="yes" Source="$(var.sourceDir)\ServiceManagement\Azure\Compute\Microsoft.WindowsAzure.Commands.ServiceManagement.PlatformImageRepository.dll" />
                         </Component>
-                        <Component Id="cmp9D64A1111722F25BC100121E73BE039E" Guid="*">
```

Comment:
@neetus15 any reasons these items are removed from the installer. This is not related to your change.
We are really warry of installer changes, most of the time your additions should not be removing any of the other components from the installer.
Please make sure you have a clean build before you generate the installer.
git status (make sure this is clean)
git clean -xdf
from the root
msbuild build.proj
After a successful build, please run Generate.ps1 -buildConfig Debug